pkg/core/interop_system_test.go

```diff
@@ -518,8 +522,18 @@ func getTestContractState(bc *Blockchain) (*state.Contract, *state.Contract) {
 			},
 			ReturnType: smartcontract.VoidType,
 		},
+		{
+			Name:   "refuelGas",
+			Offset: refuelOff,
+			Parameters: []manifest.Parameter{
+				manifest.NewParameter("account", smartcontract.Hash160Type),
+				manifest.NewParameter("gasRefuel", smartcontract.IntegerType),
+				manifest.NewParameter("gasBurn", smartcontract.IntegerType),
```

Comment:
Why three parameters here?

Reply:
It is custom contract method. The point of `gasBurn` is to burn gas after the refuelling to ensure than fueled GAS was available for the execution immediately.